packages/grid/x-data-grid/src/hooks/features/export/useGridPrintExport.tsx

```diff
@@ -54,6 +62,7 @@ export const useGridPrintExport = (
   const updateGridColumnsForPrint = React.useCallback(
     (fields?: string[], allColumns?: boolean) =>
       new Promise<void>((resolve) => {
+        // TODO remove unused Promise
```

Comment:
Are those `TODO` here because it's breaking ?
If so, we started the alpha phase so we can do the breaking change I think.

Reply:
> Are those TODO here because it's breaking?

The TODO is here because it would make it harder to review what changed in the PR.

Reply:
Make sense
We can make a follow up after that to remove the promise then